app/views/handlers/feedback.py

```diff
@@ -58,7 +62,23 @@ def get_page_title(self):
         return self.PAGE_TITLE
 
     def handle_post(self):
-        self._session_store.session_data.feedback_count += 1
+        session_data = self._session_store.session_data
+        session_data.feedback_count += 1
+        feedback_upload = FeedbackUpload(
+            session_data.feedback_count,
+            self._schema.form_type,
+            session_data.language_code,
+            self._schema.region_code,
+            session_data.tx_id,
+            self.form.data,
+        )
+        feedback_upload = current_app.eq["feedback_submitter"].upload(
```

Comment:
The same variable name (`feedback_upload`) should not be used for something else here.